tensorflow/python/data/experimental/ops/error_ops.py

```diff
@@ -41,26 +40,30 @@ def ignore_errors():
   dataset =
       dataset.apply(tf.data.experimental.ignore_errors())  # ==> {1., 0.5, 0.2}
   ```
+  Args:
+     log_warning: (Optional.) A 'tf.bool' scalar indicating whether ignored
+      errors should be logged to stderr. Defaults to 'False'.
 
   Returns:
     A `Dataset` transformation function, which can be passed to
     `tf.data.Dataset.apply`.
   """
 
   def _apply_fn(dataset):
-    return _IgnoreErrorsDataset(dataset)
+    return _IgnoreErrorsDataset(dataset, log_warning)
 
   return _apply_fn
 
 
 class _IgnoreErrorsDataset(dataset_ops.UnaryUnchangedStructureDataset):
   """A `Dataset` that silently ignores errors when computing its input."""
 
-  def __init__(self, input_dataset):
+  def __init__(self, input_dataset, log_warning):
     """See `Dataset.ignore_errors()` for details."""
     self._input_dataset = input_dataset
     variant_tensor = (
         gen_experimental_dataset_ops.ignore_errors_dataset(
```

Comment:
This needs to be under a forward compatibility guard set three weeks from today. See https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/compat/compat.py

Reply:
Fixed.